content/en/users/dev-env/notebooks/integration/_index.md

```diff
@@ -108,6 +108,15 @@ print(execution.status)
 Note that inputs that point to a URL should be specified using the
 `ComplextDataInput` class as shown above.
 
+## EGI-WMS (DIRAC)
```

Comment:
```suggestion
## EGI Workload Manager (DIRAC)
```

Could you also move this section to before the D4Science one?
Also add a reference to the DIRAC section to the docs

Reply:
Added all suggestions.

Reply:
Could you also tackle the other comments?
- Move this section to before the D4Science one?
- Add a reference to the DIRAC section to the docs

Thanks!